src/operun/contactform/locales/de/LC_MESSAGES/operun.contactform.po

```diff
@@ -0,0 +1,159 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
+"POT-Creation-Date: 2019-04-26 11:32+0000\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI +ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=1; plural=0\n"
+"Language-Code: en\n"
+"Language-Name: English\n"
+"Preferred-Encodings: utf-8 latin1\n"
+"Domain: DOMAIN\n"
+
+#: operun/contactform/configure.zcml
+msgid "Installs the operun Contact-Form Add-On."
+msgstr "Installiert das operun Contact-Form Add-On."
+
+#: operun/contactform/configure.zcml
+msgid "Uninstalls the operun Contact-Form Add-On."
+msgstr "Deinstalliert das operun Contact-Form Add-On."
+
+#. Default: "Something went wrong, please contact the site administrator."
+#: operun/contactform/browser/contactform.py
+msgid "contact_sendmail_error"
+msgstr "Etwas ist schief gelaufen. Bitte wenden Sie sich an den Site-Administrator."
+
+#. Default: "Something went wrong, please check the input!"
+#: operun/contactform/browser/contactform.py
+msgid "contact_validation_error"
+msgstr "Etwas ist schief gelaufen. Bitte überprüfen Sie die Eingabe!"
+
+#. Default: "Company"
+#: operun/contactform/browser/mail_templates/mailto_admin.pt
+#: operun/contactform/browser/templates/contactform.pt
+msgid "contactform_company"
+msgstr "Firma"
+
+#. Default: "You acknowledge and accept our privacy policy."
+#: operun/contactform/browser/templates/contactform.pt
+msgid "contactform_dsgvo_checkbox_label"
+msgstr "Datenschutzerklärung zur Kenntnis genommen und akzeptiert."
+
+#. Default: "By checking this box, you understand and agree that this form collects your name, company, phone and E-Mail, so we can correspond with you."
+#: operun/contactform/browser/templates/contactform.pt
+msgid "contactform_dsgvo_message"
+msgstr "Wenn Sie dieses Kontrollkästchen aktivieren, erklären Sie sich damit einverstanden, dass dieses Formular Ihren Namen, Ihr Firma, Ihre Telefonnummer und Ihre E-Mail-Adresse erfasst, damit wir mit Ihnen korrespondieren können."
```

Comment:
`Ihr Firma` should be `Ihre Firma`

Reply:
Or instead of `dass dieses Formular Ihren Namen, Ihr Firma, Ihre Telefonnummer und Ihre E-Mail-Adresse erfasst,` just: `dass dieses Formular die eingegebenen Daten erfasst,`